app/router.js

```diff
@@ -52,6 +52,8 @@ Router.map(function () {
     this.route('payment-sent');
   });
 
+  this.route('not-found', { path: '/*path' });
```

Comment:
I'm not sure if it matters. I thought one needed to list the 404 route as the very last route in `app/router.js` (since it's a catch-all).

I ran your branch locally and tried to find a failing case if we continue to list the 404 route somewhere in-between. However, I couldn't find a case.

Reply:
I would be surprised if it does, I couldn't find anything in the docs as well. I added the route here to keep everything sorted alphabetically. I can still move it towards the bottom if you want? Just to be sure.

Reply:
I prefer listing routes alphabetically to help with maintenance. If we later find that a URL is incorrectly considered as `not-found`, we can open an issue then.